int16 ir_0
int16 ir_1
int16 ir_2
int16 ir_3
int16 ir_4
